starter-code/index.html

```diff
@@ -4,39 +4,68 @@
   <meta charset="UTF-8">
   <title>Spotify</title>
 </head>
+<link rel="stylesheet" href="./css/main.css">
 <body>
-Premium Discover Help Download
+<header>
+  <navbar>
```

Comment:
`<navbar>` is not a real tag? 
![image](https://user-images.githubusercontent.com/29879270/38563728-9127edf0-3cab-11e8-8b11-5c196f6816e5.png)


I think you meant to create the semantic `<nav>` tag.
OR 
I think you meant to create a nav class, so `<nav class="navbar">`